tests/e2e/utils/CheReporter.ts

```diff
@@ -178,6 +180,14 @@ class CheReporter extends mocha.reporters.Spec {
 			const browserLogsStream: WriteStream = fs.createWriteStream(browserLogsFileName);
 			browserLogsStream.write(Buffer.from(browserLogs));
 			browserLogsStream.end();
+
+			// take networking logs and write to file
+			const networkLogsEntries: logging.Entry[] = await this.driverHelper.getDriver().manage().logs().get('performance');
```

Comment:
I noticed `this.driverHelper` is `undefined` when this code is reached. It seems `driverHelper` is not being injected? Or maybe it's something else>

I am running the tests with 
```
export USERSTORY=SmokeTest
export MOCHA_DIRECTORY=.
npm run test
```

and I'm forcing a failure by adding
```
expect(false).to.be.true;
```

inside one of the smoke tests.

Reply:
Do you have any ideas @ScrewTSW @nallikaea ?

Reply:
@dkwon17 
MOCHA_DIRECTORY - set up to run spec.ts files from specific directory (see pic), for example ./spec/api
Unnecessary to set it by default run
![image](https://github.com/eclipse/che/assets/72726250/84027df0-ee4f-4de3-9c98-94350f7b1796)

`export USERSTORY=SmokeTest && npm run test` should be enough. If any problems, could you attach full log, please?

Reply:
I don't have any issues with running the smoke tests regularly on the main branch, as everything passes on my machine: [logs](https://gist.github.com/dkwon17/4c75b9d8ef9de337d9792f4946026d71) 

But, when I try to introduce a test failure to test this PR:
```
--- a/tests/e2e/specs/SmokeTest.spec.ts
+++ b/tests/e2e/specs/SmokeTest.spec.ts
@@ -45,6 +45,7 @@ suite('The SmokeTest userstory', function (): void {
                });
                test('Check a project folder has been created', async function (): Promise<void> {
                        const projectName: string = FACTORY_TEST_CONSTANTS.TS_SELENIUM_PROJECT_NAME || StringUtil.getProjectNameFromGitUrl(factoryUrl);
+                       expect(false).to.be.true;
                        projectSection = (await new SideBarView().getContent().getSections())[0]; // get the (WORKSPACE) section from the sidebar - contains project content
                        expect(await projectSection.findItem(projectName)).not.eqls(undefined);
                });
```

the test fails as expected but, the [screenshot, pagesource, and browserlog](https://github.com/eclipse/che/blob/74f10465b196d7b752c68c50b8d69b10135e8750/tests/e2e/utils/CheReporter.ts#L140-L142) is not generated in the `report` folder:
```
dakwon$ pwd
/Users/dakwon/Documents/git/che/tests/e2e/report/The_SmokeTest_userstory_Create_workspace_from_factory-https-++github.com+che-incubator+quarkus-api-example.git_Check_a_project_folder_has_been_created

dakwon$ ls -l
total 0
```

I created an issue here https://github.com/eclipse/che/issues/22529, is there something I'm doing wrong?